operations/bills.md

```diff
@@ -209,9 +209,9 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
 
 #### Bill type
 
-A bill is either a `Receipt` which means that it has been fully paid, or `Invoice` that is supposed to be paid in the future.
+In default, a bill is `Receipt` indicating that it has been partially or fully paid or has not yet been paid. Alternatively, it may be `Invoice` that is supposed to be paid in the future.
```

Comment:
Did you ask the domain owners to explain the difference between these two types?

When looking at the source code, it seems that the concept of Invoice is more concrete while Receipt is a default type used for everything else. Also, I found [this validation rule](https://github.com/MewsSystems/mews/blob/48f21667d6ad97f2492f0829b44f14c4633cecca/src/Modules/Core/Mews.Business/Finance/Documents/Bills/BillComponent.cs#L541) which says that only invoices can be marked as paid. I think we don't need to base the distinction between bill types on when it is paid, there must be something else.